### ISSUE TITLE
ggplot incompatible with latest pandas?

### ISSUE THREAD

**Issue by cswarth** [yhat/ggplot#428]
It looks like the API for pandas pivot tables has changed breaking construction of histograms in python ggplot.   In Pandas 16.1, `pivot_table()` does not take a `rows=` keyword argument.  This breaks `stat_bin:_calculate()` in `ggplot/stats/stat_bin.py`

```
import pandas as pd
import ggplot
print("ggplot version {}".format(ggplot.__version__))
df = pd.DataFrame({'x':[2,5,5]})
p = ggplot.ggplot(ggplot.aes(x='x'), data=df) + ggplot.geom_histogram(binwidth=1)
print(p)

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-92a3157490f0> in <module>()
      6 df = pd.DataFrame({'x':[2,5,5]})
      7 p = ggplot.ggplot(ggplot.aes(x='x'), data=df) + ggplot.geom_histogram(binwidth=1)
----> 8 print(p)

/home/matsengrp/local/matsenv/lib/python2.7/site-packages/ggplot/ggplot.pyc in __repr__(self)
    109     def __repr__(self):
    110         """Print/show the plot"""
--> 111         figure = self.draw()
    112         # We're going to default to making the plot appear when __repr__ is
    113         # called.

/home/matsengrp/local/matsenv/lib/python2.7/site-packages/ggplot/ggplot.pyc in draw(self)
    305 
    306                     data = self._make_plot_data(data, _aes)
--> 307                     callbacks = geom.plot_layer(data, ax)
    308                     if callbacks:
    309                         for callback in callbacks:

/home/matsengrp/local/matsenv/lib/python2.7/site-packages/ggplot/geoms/geom.pyc in plot_layer(self, data, ax)
    115         _cols = set(data.columns) & set(self.manual_aes)
    116         data = data.drop(_cols, axis=1)
--> 117         data = self._calculate_stats(data)
    118         self._verify_aesthetics(data)
    119         _needed = self.valid_aes | self._extra_requires

/home/matsengrp/local/matsenv/lib/python2.7/site-packages/ggplot/geoms/geom.pyc in _calculate_stats(self, data)
    276                 new_data = new_data.append(_data, ignore_index=True)
    277         else:
--> 278             new_data = self._stat._calculate(data)
    279 
    280         return new_data

/home/matsengrp/local/matsenv/lib/python2.7/site-packages/ggplot/stats/stat_bin.pyc in _calculate(self, data)
    125                             })
    126         _wfreq_table = pd.pivot_table(_df, values='weights',
--> 127                                       rows=['assignments'], aggfunc=np.sum)
    128 
    129         # For numerical x values, empty bins get have no value

TypeError: pivot_table() got an unexpected keyword argument 'rows'
```

**Comment by cswarth** [comment]
Ran across this comment in `pandas/doc/source/whatsnew/v0.16.0.txt`

```
DataFrame.pivot_table and crosstab's rows and cols keyword arguments were removed in favor
+  of index and columns (:issue:6581)
```

**Comment by Arn-O** [comment]
``geom_bar()`` is also impacted.

**Comment by jaycode** [comment]
Please merge my fork, or while waiting, you can install from:

```
pip install git+git://github.com/jaycode/ggplot.git
```

**Comment by glamp** [comment]
fixed in 0.9.3
